pyworkflow/utils/properties.py

```diff
@@ -387,6 +387,16 @@ class PluginStates:
     ERRORS = 'errors'
 
 
+class PluginInformation:
```

Comment:
What is all this? This seems to belong to a different task.

Reply:
I see this is needed by scipion-app?

Reply:
It's solved!!